dashboard/src/components/Table/TreeTable.tsx

```diff
@@ -13,9 +13,9 @@ import {
   possibleTestsTableFilter,
 } from '@/types/tree/TreeDetails';
 
-import { TableRow, TableCell } from '../ui/table';
+import { TreeTableBody, zOrigin } from '@/types/tree/Tree';
 
-import { TreeTableBody, zOrigin } from '../../types/tree/Tree';
+import { TableRow, TableCell } from '../ui/table';
```

Comment:
since you are changing here, can you update for `@` import?
the reason we are allowing `..` in this project is because we didn't have the plugin for @ imports on vite